sdk/python/tests/compiler/testdata/conditions_and_loops.py

```diff
@@ -64,8 +64,8 @@ def produce_number() -> int:
 
 
 @dsl.pipeline(name='Conditions and loops')
-def conditions_and_loops(n='3', threshold='20'):
-    produce_numbers_task = produce_numbers(n)
+def conditions_and_loops(n1='3', threshold='20'):
```

Comment:
json parser fails with name as `n`.

Reply:
I think this is a bug, we should have double quote on single character parameter for n and y to avoid this issue. For now I will lgtm and open another issue for it